packages/back-end/src/jobs/sdkWebhooks.ts

```diff
@@ -320,20 +328,27 @@ export async function fireSdkWebhook(
 ) {
   const webhookContext = getContextForAgendaJobByOrgObject(context.org);
 
+  const sendPayload =
+    webhook.httpMethod !== "GET" &&
+    sendPayloadFormats.includes(webhook.payloadFormat ?? "standard");
+
+  if (!sendPayload) return;
```

Comment:
This will break.

On main, if `sendPayload` is false, it still sends the webhook, it just sets the payload to an empty string.  This PR skips sending the webhook entirely, which is wrong.